frontend/lib/util/date_utils.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:fixnum/fixnum.dart';
+
+Int64 secondsSinceEpoch(DateTime date) {
+  return Int64(date.difference(DateTime(1970)).inSeconds);
```

Comment:
I think it would be better to use (date.toUtc().millisecondsSinceEpoch / 1000).round() or similar. I think DateTime(1970) creates a datetime in local time zone, but not sure.

Reply:
this should also do the trick `DateTime(1970).toUtc()` without any further calculation?

Reply:
I think not: DateTime(1970).toUtc() would create a DateTime at 1.1.1970 00:00 **local time zone** and then convert it to UTC, resulting in, depending on the local time zone, 31.12.1969 22:00 UTC. You can however use `DateTime.utc(1970)` to initialize a date at UTC timezone. So the following would work:
```dart
  return Int64(date.difference(DateTime.utc(1970)).inSeconds)
```